app/pages/index.tsx

```diff
@@ -25,13 +28,26 @@ const StyledBtnContainer = styled('div')`
   margin: 24px 0;
 `;
 
+// while a `strong` element may be displayed as bold,
+// that should not be relied upon and font-weight should be used instead.
+// Using a `strong` element is still useful on a semantic level.
+const BoldText = styled('strong')`
+  font-weight: bold;
+`;
+
+const StyledCallout = styled(Callout)`
+  margintop: 16px;
+  marginbottom: 16px;
```

Comment:
```suggestion
  margin-top: 16px;
  margin-bottom: 16px;
```

Or just `margin: 16px 0` would be better. Though I don't believe this is will actually do anything anyways as it is already the default element style:

![margin](https://user-images.githubusercontent.com/14259474/195692511-3c6588c7-d878-4fe6-9d6f-69393ea01682.png)

Reply:
Good catch! This is a half-baked refactor :sweat_smile:

Reply:
I'd say so, it is cleaner and ensures the value of margin on both sides is identical. Though if we were using a global margin variable to ensure they were coming from the same constant I could see why you would think it's easier to read.

Reply:
I was hesitating between multiple options. Now there's both the shorthand property, and a `spacing` section in our theme.

Reply:
Love it, nice job.